seeders/20180219074913-ActionsSeed.js

```diff
@@ -6,8 +6,58 @@ module.exports = {
     return queryInterface.bulkInsert('Actions', [
 
       { name: "HTML boilerplate", url: "https://www.sitepoint.com/introduction-html5-boilerplate/" , criterionId: 10000, createdAt: new Date(), updatedAt: new Date(), id: 10000 },
-      { name: "CSS", url: "https://github.com/twbs/bootstrap" , criterionId: 10000, createdAt: new Date(), updatedAt: new Date(), id: 10002 },
+      { name: "Writing HTML", url: "https://www.w3schools.com/html/" , criterionId: 10001, createdAt: new Date(), updatedAt: new Date(), id: 10001 },
+      { name: "Writing HTML", url: "https://www.w3schools.com/html/html_basic.asp" , criterionId: 10001, createdAt: new Date(), updatedAt: new Date(), id: 10002 },
+      { name: "Writing HTML", url: "https://www.codecademy.com/courses/web-beginner-en-HZA3b/0/1?curriculum_id=50579fb998b470000202dc8b" , criterionId: 10002, createdAt: new Date(), updatedAt: new Date(), id: 10003 },
+      { name: "Writing HTML", url: "https://www.w3schools.com/html/html_styles.asp" , criterionId: 10002, createdAt: new Date(), updatedAt: new Date(), id: 10004 },
+      { name: "Writing Templates", url: "https://www.impressivewebs.com/w3schools-ugly-bad-good/" , criterionId: 10003, createdAt: new Date(), updatedAt: new Date(), id: 10005 },
```

Comment:
Okay man, I don't think you're trying here.  This has absolutely nothing to with the criterion.  It's a review of the W3 school's website and coding quality.  The criteria here is "Uses HTML tags semantically, e.g. uses structural tags (section, header, footer, article, etc.)".   So google `semantic HTML tags` and you should find resources like this... 

https://en.wikipedia.org/wiki/Semantic_HTML
https://internetingishard.com/html-and-css/semantic-html/
https://www.lifewire.com/why-use-semantic-html-3468271

Cmon man.